core/src/main/java/nl/nn/adapterframework/pipes/UnzipPipe.java

```diff
@@ -271,73 +269,52 @@ public PipeRunResult doPipe(Message message, PipeLineSession session) throws Pip
 	public void setDirectory(String string) {
 		directory = string;
 	}
-	public String getDirectory() {
-		return directory;
-	}
 
 	@IbisDoc({"2", "Sessionkey with a directory value to extract the archive to", ""})
 	public void setDirectorySessionKey(String directorySessionKey) {
 		this.directorySessionKey = directorySessionKey;
 	}
-	public String getDirectorySessionKey() {
-		return directorySessionKey;
-	}
 
 	@IbisDoc({"3", "If true, file is automatically deleted upon normal JVM termination", "true"})
 	public void setDeleteOnExit(boolean b) {
 		deleteOnExit = b;
 	}
-	public boolean isDeleteOnExit() {
-		return deleteOnExit;
-	}
 
 	@IbisDoc({"4", "If set <code>false</code>, only a small summary (count of items in zip) is returned", "true"})
 	public void setCollectResults(boolean b) {
 		collectResults = b;
 	}
-	public boolean isCollectResults() {
-		return collectResults;
-	}
 
 	@IbisDoc({"5", "If set <code>true</code>, the contents of the files in the zip are returned in the result xml message of this pipe. Please note this can consume a lot of memory for large files or a large number of files", "false"})
 	public void setCollectFileContents(boolean b) {
 		collectFileContents = b;
 	}
-	public boolean isCollectFileContents() {
-		return collectFileContents;
-	}
 
 	@IbisDoc({"6", "Comma separated list of file extensions. Files with an extension which is part of this list will be base64 encoded. All other files are assumed to have UTF-8 when reading it from the zip and are added as escaped xml with non-unicode-characters being replaced by inverted question mark appended with #, the character number and ;", "false"})
 	public void setCollectFileContentsBase64Encoded(String string) {
 		collectFileContentsBase64Encoded = string;
 	}
-	public String getCollectFileContentsBase64Encoded() {
-		return collectFileContentsBase64Encoded;
-	}
 
 	@IbisDoc({"7", "If set <code>false</code>, a suffix is added to the original filename to be sure it is unique", "false"})
 	public void setKeepOriginalFileName(boolean b) {
 		keepOriginalFileName = b;
 	}
-	public boolean isKeepOriginalFileName() {
-		return keepOriginalFileName;
-	}
 
-	@IbisDoc({"8", "If set <code>true</code>, subdirectories in the zip file are created in the directory to extract the archive to", "false"})
-	public void setCreateSubdirectories(boolean b) {
-		createSubdirectories = b;
+	@Deprecated
+	@ConfigurationWarning("the attribute 'createSubDirectories' has been renamed to 'keepOriginalFilePath'")
+	public void setCreateSubDirectories(boolean b) {
+		keepOriginalFilePath = b;
```

Comment:
When you replace a setter, like this, then call the new setter from the replaced one, instead of copying the contents